.github/actions/setup/action.yaml

```
@@ -21,7 +21,11 @@ runs:
         LIBSECCOMP_LIB_PATH: /usr/local/libseccomp/lib
       run: |
         install_dir=$(dirname ${LIBSECCOMP_LIB_PATH})
-        sudo ./scripts/install_libseccomp.sh -m -v ${{ inputs.version }} -i ${install_dir}
+        args=""
+        if [[ "${{ inputs.link-type }}" == "static" ]]; then
+            args="-m"
+        fi
+        sudo ./scripts/install_libseccomp.sh ${args} -v ${{ inputs.version }} -i ${install_dir}
```

```suggestion
        args=()
        if [[ "${{ inputs.link-type }}" == "static" ]]; then
            args+=("-m")
        fi
        sudo ./scripts/install_libseccomp.sh "${args[@]}" -v ${{ inputs.version }} -i ${install_dir}
```

Untested, but I would use an array rather relaying on word-splitting.

---

I have habit of using an empty variable instead of an empty array because bash treats an empty array as an undeclared variable, so the script fails with `set -u`. But this action doesn't set `set -u`, so array would be better.